File: Day 9/secretNumber.cpp
#include <iostream>
#include <math.h>
#include <cstdlib>

using namespace std;

int main()
{

    int secretN = rand() % 11;
    int n;
    int tries = 0;

    do{

        cout << "Try to guess the secret Number! " << endl;
        cin >> n;
        
        if( n == secretN )
        {
            cout << "Congratulations! You win! " << endl;
        } 
        else if( n > secretN)
        {
            cout << "The secret number is smaller than your number. Try again! " <<endl;
            tries++;
        }
        else
        {
            cout << "The secret number is higher than your number. Try again! " <<endl;
            tries++;
        }

    } while( n != secretN );



    return 0;
}

File: Day 9/secretNumber.js
function aleatorio(min, max)
{
    var res;
    res = Math.floor(Math.random() * (max - min +1)) + min;
    return res;
}

let secretN = aleatorio(0, 10)

let tries = 0
let n

do{

     n = prompt('Try to guess the secret Number!')
    
    if( n == secretN )
    {
        alert('Congratulations! You win!')
    } 
    else if( n > secretN)
    {
        alert('The secret number is smaller than your number. Try again!' )
        tries++
    }
    else
    {
        alert('The secret number is higher than your number. Try again!' )
        tries++
    }

} while( n != secretN )